infrastructure/modules/armonik/external-data.tf

```diff
@@ -76,7 +76,7 @@ locals {
   })
 
   admin_api_url = "http://${local.admin_gui_endpoints.ip}:${local.admin_gui_endpoints.api_port}/api"
-  admin_app_url = "http://${local.admin_gui_endpoints.ip}:${local.admin_gui_endpoints.app_port}/"
```

Comment:
You shouldn't need to remove the end slash

Reply:
Apparently, proxy_pass, when used in a location using regex, doesn't work with an URI ending by a slash.

There is more information about this here : [serverfault.com](https://serverfault.com/questions/649151/nginx-location-regex-doesnt-work-with-proxy-pass#693951)

Reply:
Even without the regex, it doesn't seems to work with the slash.

Reply:
ok, weird